core/src/banking_stage/packet_deserializer.rs

```diff
@@ -198,6 +198,19 @@ impl PacketDeserializer {
             }
         })
     }
+
+    #[allow(dead_code)]
+    pub(crate) fn deserialize_packets_with_indexes(
```

Comment:
the indexes are used for ordering in block-verification. we should be using priority instead of receive-order, right?

Reply:
yeah, eventually, but not at the time of this pr...